central/telemetry/billingmetrics/data.go

```diff
@@ -0,0 +1,69 @@
+package billingmetrics
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	bmetrics "github.com/stackrox/rox/central/billingmetrics/store"
+	cluStore "github.com/stackrox/rox/central/cluster/datastore"
+	"github.com/stackrox/rox/central/sensor/service/pipeline/clustermetrics"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/set"
+)
+
+var (
+	clusterReader = sac.AllowFixedScopes(
+		sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+		sac.ResourceScopeKeys(resources.Cluster))
+
+	metricsWriter = sac.AllowFixedScopes(
+		sac.AccessModeScopeKeys(storage.Access_READ_ACCESS, storage.Access_READ_WRITE_ACCESS),
+		sac.ResourceScopeKeys(resources.Administration))
+
+	previousMetrics = &clustermetrics.BillingMetrics{}
+)
+
+func average(metrics ...*clustermetrics.BillingMetrics) clustermetrics.BillingMetrics {
+	n := int64(len(metrics))
+	a := clustermetrics.BillingMetrics{}
```

Comment:
a -> aggregatedMetrics

Declaring it like the following is more idiomatic
```
var aggregatedMetrics clustermetrics.BillingMetrics
```

Also typically you want to pass protos around by reference so the return should be *clustermetrics.BillingMetrics

Reply:
Refactored in #7137.